src/util.js

```diff
@@ -495,10 +495,24 @@ Candy.Util = (function(self, $){
 		 *   Emotified text
 		 */
 		emotify: function(text) {
-			var i;
-			for(i = this.emoticons.length-1; i >= 0; i--) {
+			var codeMatches = text.match(/(\`[^`]*\`)/gim) || [];
+			var codeHolders = {
```

Comment:
At first I was using `addCodeHolders` and `replaceCodeHolders`, then I refactored it to be like that but I found weird as well.

I thought about moving that code outside of that function, but I didn't want to expose that.

Do you have a better suggestion?